practice-two/src/components/FormValidation/FormValidation.jsx

```diff
@@ -0,0 +1,180 @@
+import { useState } from 'react';
+import './FormValidation.css';
+import { InputValidate, Button } from '@components';
+import { BUTTON_VARIANTS } from '@constants';
+import { Validation } from '@helpers';
+
+const FormValidation = () => {
+  const [formData, setFormData] = useState({
+    username: '',
+    avatar: '',
+    email: '',
+    phonenumber: '',
+    description: '',
+    address: '',
+    gender: 'male',
+  });
+
+  const [errors, setErrors] = useState({
+    username: '',
+    avatar: '',
+    email: '',
+    phonenumber: '',
+    description: '',
+    address: '',
+  });
+
+  const onChange = (e) => {
+    setFormData({ ...formData, [e.target.name]: e.target.value });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    const newErrors = {};
+    for (const field in formData) {
+      const errorMessage = Validation(field, formData[field]);
+      if (errorMessage) {
+        newErrors[field] = errorMessage;
+      }
+    }
+    setErrors(newErrors);
+
+    if (Object.keys(newErrors).length === 0) {
+      console.log(formData);
```

Comment:
At first, I create a variable newErrors with empty object
It will store any error message when Validation function loop through each input field

After that, I use Object.keyt to retrieve an array of strings representing all the keys in an object
Check if the array's length is 0, the form can submit